packages/discord.js/src/structures/InteractionResponse.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const { InteractionType } = require('discord-api-types/v9');
+
+/**
+ * Represents an interaction's response
+ */
+class InteractionResponse {
+  /**
+   * @param {Interaction} interaction The interaction associated with this response
+   * @param {Snowflake?} responseInteractionId The interaction id associated with the original response
+   * @private
+   */
+  constructor(interaction, responseInteractionId) {
+    /**
+     * The interaction associated with the interaction response
+     * @type {Interaction}
+     */
+    this.interaction = interaction;
+    /**
+     * The id of the original interaction response
+     * @type {Snowflake}
+     */
+    this.responseInteractionId = responseInteractionId ?? interaction.id;
```

Comment:
`message.interactionResponse.responseInteractionId` sounds... funky, because it contains "interaction response" and then the same but backwards ("response interaction").

Plus the class is also called `InteractionResponse`, so I believe this field should be named just `id`, as it's an interaction response ID.